mpas_analysis/shared/interpolation/interpolate.py

```diff
@@ -0,0 +1,68 @@
+"""
+    Module that provides basic nearest neighbor interpolation functionality
+    
+    Author: Luke Van Roekel
+    Modified: 10/24/2016
+"""
+
+import numpy as np
+from scipy.spatial import cKDTree
+import sys
+
+def lon_lat_to_cartesian(lon, lat, R = 6371222):
```

Comment:
R should probably be a float (6371222.) instead of an integer.  Sometimes weird things happen when you use integers that should be floats.